parquet-hadoop/src/main/java/org/apache/parquet/hadoop/InternalParquetRecordWriter.java

```diff
@@ -102,6 +103,8 @@ private void initStore() {
     MessageColumnIO columnIO = new ColumnIOFactory(validating).getColumnIO(schema);
     this.recordConsumer = columnIO.getRecordWriter(columnStore);
     writeSupport.prepareForWrite(recordConsumer);
+    System.out.println(String.format("Created ParquetWriter with [%d, %d] for block size checks. Estimation(%s). BlockSize(%d)",
```

Comment:
Please use SLF4J for logging. You can see examples in the rest of the library.

Reply:
Oops... missed removing this statement.